sdcm/cluster.py

```diff
@@ -2170,10 +2170,16 @@ def web_install_scylla(self, scylla_version: Optional[str] = None) -> None:
             f"curl -sSf get.scylladb.com/server | sudo bash -s -- --scylla-version {version}")
 
     def install_scylla_debuginfo(self) -> None:
+        if ComparableScyllaVersion(self.scylla_version) > '2025.1.0~dev':
```

Comment:
If to be really correct, then it should be `bigger or equal` (`>=`).
But practically, who is going to run dev version of the 2025.1?